examples/tiny-yolo-v2-coco/README.md

```diff
@@ -0,0 +1,14 @@
+# Tiny YOLO v2 #
+
+This is an example of Tiny YOLO v2 neural network. You can read about this network [here](https://pjreddie.com/darknet/yolov2/).
+
+Folder `model` contains yolov2-tiny.cfg on which file `feedforward.go` based.
+
+Folder `data` contains image file `dog_416x416.jpg` - this is scaled to 416x416 image for make it better understanding of how net works.
+
+For now this example works not quite fast (it is only "tiny" version). Probably it is better to implement this network using CUDA.
```

Comment:
Is it possible to attach a bench or a comparison with another implementation CPU based? It would be very interesting for the reader to have a fact; people love to compare :D

Reply:
Oh. I would like to, but have not that much time right now (can work at week-ends).
I guess in future I'll make comparison of pure darknet implementation (C lang), darkflow (tensorflow, python), this (Golang), and maybe OpenCV (they have boosted C++ version).

Reply:
Cool